repository/src/lib.rs

```diff
@@ -103,9 +124,10 @@ impl<T: RawRepository> DistributedRepository<T> {
     /// Checks the validity of the repository, starting from the given height.
     ///
     /// It checks
-    /// 1. all the reserved branches and tags
+    /// 1. all the reserved branches and tags (especially valiity of the `fp` branch)
```

Comment:
What do you mean `especially` here? 
Do you wanted to say that we always have to check the validity of `fp` branch?

Reply:
It is just emphasized as it's newly added. I updated the description.